docs/api/cli/scaffold_fsm.md

```diff
@@ -212,61 +243,31 @@ Get the file content.
 ## ModelsFileGenerator Objects
 
 ```python
-class ModelsFileGenerator(AbstractFileGenerator)
+class ModelsFileGenerator(SimpleFileGenerator,  MODELS)
```

Comment:
Why are we adding docs of this kind. Not just here but generally, who needs them? Essentially they are a pretty copy of the code. Are they not just adding noise to the docs?

Reply:
no we don't need them for this. They are simply automatically generated, one could exclude specific folders and files from being included I'm sure.